integration-tests/environments/react-native-test-app/package.json

```diff
@@ -0,0 +1,150 @@
+{
+  "name": "@realm/react-native-test-app-tests",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "build:android": "mkdirp dist/res && react-native bundle --entry-file index.js --platform android --dev true --bundle-output dist/main.android.jsbundle --assets-dest dist/res",
+    "build:ios": "mkdirp dist && react-native bundle --entry-file index.js --platform ios --dev true --bundle-output dist/main.ios.jsbundle --assets-dest dist",
```

Comment:
```suggestion
    "build:ios": "mkdir dist && react-native bundle --entry-file index.js --platform ios --dev true --bundle-output dist/main.ios.jsbundle --assets-dest dist",
```

Reply:
This was added by the RNTA template - it will invoke [mkdirp](https://www.npmjs.com/package/mkdirp), we could probably leave this out for now, since we don't expect to run the app outside of a simulator. But I wanted to keep this as close to the template as possible.

Reply:
FWIW, if you generate a new project today, we no longer rely on `mkdirp`. Instead, we use an inline script `mkdist`:
```json
  "scripts": {
    "android": "react-native run-android",
    "build:android": "npm run mkdist && react-native bundle --entry-file index.js --platform android --dev true --bundle-output dist/main.android.jsbundle --assets-dest dist/res",
    "build:ios": "npm run mkdist && react-native bundle --entry-file index.js --platform ios --dev true --bundle-output dist/main.ios.jsbundle --assets-dest dist",
    "build:macos": "npm run mkdist && react-native bundle --entry-file index.js --platform macos --dev true --bundle-output dist/main.macos.jsbundle --assets-dest dist",
    "build:windows": "npm run mkdist && react-native bundle --entry-file index.js --platform windows --dev true --bundle-output dist/main.windows.bundle --assets-dest dist",
    "ios": "react-native run-ios",
    "lint": "eslint .",
    "macos": "react-native run-macos --scheme Example",
    "mkdist": "node -e \"require('node:fs').mkdirSync('dist', { recursive: true, mode: 0o755 })\"",
    "start": "react-native start",
    "test": "jest",
    "windows": "react-native run-windows --sln windows/Example.sln"
  },
```

Reply:
We did upstream a fix. I don't remember which versions have it. We kept it in the template because some users downgrade a lot for testing purposes.

Reply:
If you don't plan to use older versions of RN, you can probably remove them 👍